en/developer_guide/fieldType.rst

```diff
@@ -0,0 +1,101 @@
+Field type
+==========
+
+Open Orchestra offers a RAD approach to create the Content Types.
+Further information about it are available in the `Content Types documentation`_.
+
+During creating a content type with different fields, you can choose between different types.
+By default Open Orchestra offers few types (text line, date, email, hidden, integer, choice, media, money, rich test, text area).
+
+
+.. image:: ../../images/fieldType.png
+
+Add a field type
+----------------
+To add a new field type you can add in your configuration parameters an entry
+to the ``open_orchestra_backoffice.field_types`` key
+which describes your field type.
+
+The description of your field type must contains these values:
+
+* ``name`` name of your field type
+* ``type`` type of your field, this can be a type natively available in Symfony or a custom type
+* ``label`` translation key of label
+* ``default_value`` (optional) describes the default value field
+* ``options`` (optional) describes options of your field type
+
+``default_value`` and ``options`` are describe in more detail in subsequent sections.
```

Comment:
I think at this point the reader should already have an idea of what is an option and what is the default_value setting.
On that version it's not the case.
So I suggest to write a little sentence for each bullet point. You can still keep the two links to a more precise desciption.